### PR TITLE
Update Feature Details and Features home page

### PR DESCRIPTION
**Pull request by powermukisa**
- Update how UI is loaded in Feature Details (this should be picked up by other pages through out the app) - @Codebreake please create a trello task for this
The new way of loading UI greatly reduces null point exceptions (NPEs) because not every bit of the code is loaded at the start
- Remove unused Db Handlers
- Add UI for assiging a user and status to a feature tests
- Refactor FeatureDetails page

### CODE REVIEW

File: src/components/features/featuredetails/UpdateFeatureTestDialog.js [hunk 1]
```diff
@@ -9,9 +9,8 @@ import DialogContentText
   from "@material-ui/core/DialogContentText/DialogContentText";
 import TextField from "@material-ui/core/TextField/TextField";
 import DialogActions from "@material-ui/core/DialogActions/DialogActions";
-import UpdateFeatureByIdDbHandler
-  from "../featuresdbhandlers/UpdateFeatureByIdDbHandler";
 import {Timestamp} from "firebase";
+import {updateFeature} from "../../../store/actions/featureActions";
```
Comment (Codebreake) [inline src/components/features/featuredetails/UpdateFeatureTestDialog.js]: extra line not needed

Comment (Codebreake) [reply src/components/features/featuredetails/UpdateFeatureTestDialog.js]: All other files LGTM

**Comment by Codebreake** [comment]
> * Update how UI is loaded in Feature Details

Trello task created.

File: src/components/features/featuredetails/UpdateFeatureTestDialog.js (hunk 1 above)
Comment (powermukisa) [reply src/components/features/featuredetails/UpdateFeatureTestDialog.js]: there's no extra line here

**Pull request merged**